addons/mail_bot/models/mail_thread.py

```diff
@@ -1,12 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import models
-
-
-class MailThread(models.AbstractModel):
-    _inherit = ['mail.thread']
-
-    def _message_post_after_hook(self, message, msg_vals):
-        self.env['mail.bot']._apply_logic(self, msg_vals)
-        return super(MailThread, self)._message_post_after_hook(message, msg_vals)
```

Comment:
Note: cleanup, this had nothing to do on a generic thread override. Moved to channel.